src/DotNet.Status.Web/Controllers/GitHubHookController.cs

```diff
@@ -53,22 +59,187 @@ private async Task EnsureLabelsAsync()
             );
         }
 
+        private readonly SimpleJsonSerializer _serializer = new SimpleJsonSerializer();
```

Comment:
What is "SimpleJsonSerializer?"  Can it not stream things? Having to duplicate the entire response body into a string isn't great for perf. Since this is going to be called for every comment in any repository anywhere, perf matters.

Reply:
Its the serializer that Octokit uses that correctly handles the StringEnum things and githubs property name formats. I didn't see any streaming option.

Reply:
Ah. Probably fine then.